string instruction
---
bool success
string message
